port/omrportpriv.h

```diff
@@ -110,6 +110,8 @@ typedef struct J9CudaGlobalData {
 
 /* these port library globals are initialized to zero in omrmem_startup_basic */
 typedef struct OMRPortLibraryGlobalData {
+	BOOLEAN huge_pages_mmap_enabled;
+	uintptr_t memfd_createDL;
```

Comment:
Should these fields be under `#if defined(LINUX)` ? I have no strong opinion.
Just noticed that `PPG_huge_pages_mmap_enabled` and `PPG_memfd_createDL` are defined for Linux only.
I believe we should be consistent (here or there)

Reply:
I'm more inclined to include `#if defined(LINUX)` here

Reply:
I wondering would it better to move these fields to `OMRPortPlatformGlobals`?

Reply:
That would probably be better